include/cmn/tipcTypes.h

```diff
@@ -18,6 +18,7 @@ typedef enum TipcServiceInstance
 	s1apAppInstanceNum_c,
 	s6AppInstanceNum_c,
 	s11AppInstanceNum_c,
+	s10AppInstanceNum_c,
```

Comment:
Do we really need separate s10 container ? Can't we have same gtp container and handle all gtp messages.

Reply:
We have separated s10 container to reduce the coupling between the original with s10 support code.